data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -167,3 +167,9 @@ def load(self) -> None:
             self.SCORE_LOW_GEOJSON, driver="GeoJSON"
         )
         logger.info("Completed writing usa-low")
+
+
+class AWSGeoScoreETL(ExtractTransformLoad):
```

Comment:
Since the only thing that changes is the source data location, I wonder if subclassing is necessary?

Reply:
*shrug* I absolutely can change this cause I have no idea what I'm doing in terms of correct python.